### ISSUE TITLE
debug Inject Error: /bin/bash: line 1: helper: command not found

### ISSUE THREAD

**Issue by ibuildthecloud** [loft-sh/devpod#280]
**What happened?**

Tried the demo go project and it fails with this output

```
[19:08:42] info Creating devcontainer...
[19:08:42] debug Inject and run command: //?/C:/Program Files/DevPod/devpod-cli.exe agent workspace up --workspace-info 'H4sIAAAAAAAA/7ySXW+bPBTHv8qjcw0xBBJcbh5VaadFe2lUkUmbkKqDfSBWDUa2SVtVfPeJ0HWd1Gh3u7TOOfD7vzzDg7H3rkdBkD+DkpDD0QkjKbTUGk+ht09hYyCA4TRcRtFFmmUypCpeh2myqkO+lBjGJBJMRSQTKSCA2mhJFnLY5GW5d2RdWUq0lspyIenYG1mWwnSeHv00oRoH7cvyFcaV5RmMXgk/WIIcDt73LmfM9NQ1FvvDolH+MFToHHm3EKZlSZwuMcULrDLkFec1yiolzsWac8wwo1WC0Qpjvl5lMllTlMi04rLmFylP6ypmrRLWOFN7dg7HmqM6SX2GDtuJSxpxTxYCML1XpnPT6Opm8+n69m53WXycnkfUwx+rgyO7mz8lIfd2oDGA7dfLTbH9ti2+3xXbL9c3+2K6fWd1HANoURxUN6U4BqAkvQFqjMZOwhiAM4Odk26Uv6XeOOWNfXpj5mzhyby/aB8DEJZwkliolpzHtj8VZJmE0SqMsyJa5hHP0+gHBKDR+b0jeX7jpQ6TK3Mdpj9gQ52feHv0B8iBsf/ZJmc7axqL7X8flCbHrui4M5LNvQqFVgt6JAhAmodOG5T728/vK9Sm9qE7vFwyS5rQkWO/DtkxWsSLJQRAjyRma++HimxHntz8fonwN+JrpqpzHrWGHGrUjmBK6R9W4icAAAD//wEAAP//OqRaQtsDAAA=' --debug
[19:08:42] debug execute inject script
[19:08:42] debug Run command provider command: ${DEVPOD} helper sh -c "${COMMAND}"
[19:08:42] debug /bin/bash: line 1: helper: command not found
[19:08:42] debug done exec
[19:08:42] debug done inject
[19:08:42] debug done injecting
[19:08:42] debug Inject Error: /bin/bash: line 1: helper: command not found
EOF
```

**What did you expect to happen instead?**

**How can we reproduce the bug?** (as minimally and precisely as possible)

My `devcontainer.json`:

```
{
    "name": "...",
    ...
}
```

**Local Environment:**

* DevPod Version: \[use `devpod --version`\]
* Operating System: windows
* ARCH of the OS: AMD64

**DevPod Provider:**

* Local/remote provider: docker

**Anything else we need to know?**

**Comment by FabianKramm** [comment]
@ibuildthecloud thanks for creating this issue! This should be fixed with the new version that we release today, the problem was that if you have a local `bash` installed on windows, DevPod would prefer this rather than the in-built shell.

**Comment by FabianKramm** [comment]
@ibuildthecloud should be fixed with version [v0.1.3](https://github.com/loft-sh/devpod/releases/tag/v0.1.3)
